docs/platform/what-is-tizen/overview.md

```diff
@@ -47,6 +47,32 @@ The most important strengths that enable Tizen to move to the next level are IoT
 
   ![IoTivity for Connectivity](media/about_tizen_6.png)
 
+## AI and Machine Learning
+
```

Comment:
"NN Runtime" should be written as a third heading to make it distinguishable from other third headings under the "AI and Machine Learning" section.
Ref: In other PRs (#1615 & #1616), different authors are working on different sections under the "AI and Machine Learning" section.
```suggestion
### NN Runtime
```